.circleci/config.yml

```diff
@@ -666,6 +666,10 @@ workflows:
             -   nova-docs:
                     context:
                         - common-build-creds
+                    filters:
+                        branches:
+                            ignore:
+                                - pull/*
```

Comment:
This is for ignoring forked pull request

Reply:
It seems like we should publish docs for forked PRs too if possible to make it easier to QA external contributions before merge.

Reply:
I disagree this can be a very bad backdoor at worst it will create on main

Reply:
I agree with baggio- security first, even if it complicates forks

Reply:
Yeah, you need github credentials via environment var to make this succeed. Forked PRs don't have access to that.

Reply:
if we changed the snyk scanner to not use tokens (and just block PRs), this might work